packages/web3-core/src/web3_request_manager.ts

```diff
@@ -241,14 +241,14 @@ export class Web3RequestManager<
 						payload,
 						error as JsonRpcResponse<ResponseType, unknown>,
 						{ legacy: true, error: true },
-					),
+					)
```

Comment:
Could you please ensure prettier is working well on your machine?
```suggestion
					),
```